src/map/map.hpp

```diff
@@ -234,17 +228,8 @@ class gamemap
 protected:
 	t_translation::t_map tiles_;
 
-	/**
-	 * The size of the starting positions array is MAX_PLAYERS + 1,
-	 * because the positions themselves are numbered from 1.
-	 */
 	tstarting_positions starting_positions_;
```

Comment:
yes, i just removed the comment because it's not true anymore, tstarting_positions is a map string -> map_lcoation, but it usesd to be a map_lcoation[MAX_PLAYERS + 1] in wesnoth 1.12